src/components/formatters/test/formatter_json_rpc_test.cc

```diff
@@ -40,12 +47,47 @@
 
 namespace test {
 namespace components {
-namespace formatters {
+namespace formatters_test {
 
 using namespace NsSmartDeviceLink::NsSmartObjects;
 using namespace NsSmartDeviceLink::NsJSONHandler::Formatters;
 using namespace NsSmartDeviceLink::NsJSONHandler::strings;
 
+void CompactJson(std::string& str) {
+  Json::Value root;
+  Json::Reader reader;
+  reader.parse(str, root);
+  Json::FastWriter writer;
+  str = writer.write(root);
+  if (str[str.size() - 1] == '\n') {
+      str.erase(str.size()-1,1);
+  }
+  std::string::iterator end_pos = std::remove(str.begin(), str.end(), '\n');
```

Comment:
Why you need these two lines? They remove all endl from string. We have to remove only the last.